smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/endpointsV2/ParameterGenerator.java

```diff
@@ -76,7 +105,7 @@ public Map.Entry<String, String> getNameAndType() {
     public String toCodeString() {
         String buffer = "";
```

Comment:
nit: StringBuilder can be used here too, but this diff is not touching the buffer.

Reply:
If this is for performance and not readability, I would push back on the change. 
I believe string concat is analyzable and optimizable by current Java versions to be effectively identical to appends. 
I would also contend that this is a very small string and this code is not a bottleneck. 

If you prefer the readability of append, then I will accept the change.